### DISCUSSION TITLE
how to get the result of feature engineering

### DISCUSSION THREAD

**Discussion by Flyfoxs** [pycaret/pycaret#1970] [Q&A]
In the document, I can find the comparsion the DF between before and after as below. But not sure how to manully get the DataFrame after feature engineering by myself.

![image](https://user-images.githubusercontent.com/11268302/146632474-b504d48e-2a46-4f4f-a5b0-a7db31fc1561.png)

**Comment by virtualpeer** [comment]
mark , fellow you

**Comment by ngupta23** [comment]
@tvdboom - FYI...
